From 5c62e7afe2197c7b5bb00ed70bc6960b49a0317e Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Thu, 18 Jan 2018 19:23:56 +0100
Subject: [PATCH] units: don't put udev to its own mount namespace with slave
 propagation

Change in upstream was done mostly for political reasons to discourage
people from doing mounts in udev rules. RHEL is very bad place for
such experiments. Revert to default we shipped with RHEL-7 GA.

RHEL-only

Resolves: #1432211
---
 units/systemd-udevd.service.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/units/systemd-udevd.service.in b/units/systemd-udevd.service.in
index 32f04d901a..46b0795151 100644
--- a/units/systemd-udevd.service.in
+++ b/units/systemd-udevd.service.in
@@ -21,5 +21,4 @@ Sockets=systemd-udevd-control.socket systemd-udevd-kernel.socket
 Restart=always
 RestartSec=0
 ExecStart=@rootlibexecdir@/systemd-udevd
-MountFlags=slave
 KillMode=mixed
